File: HYPERLEDGER/bfn-network-chaincode/bfn-chaincode2/src/helpers/models/sector.ts

import { JsonObject, JsonProperty } from 'json2typescript';

@JsonObject('Sector')
export class Sector {

  @JsonProperty('docType', String)
  public docType?: string | undefined;

  @JsonProperty('sectorId', String)
  public sectorId: string | undefined;

  @JsonProperty('sectorName', String)
  public sectorName: string | undefined;

}

File: HYPERLEDGER/bfn-network-apps/businesslibrary/lib/data/sector.dart
class Sector {
  String sectorId, sectorName, description;

  Sector({this.sectorId, this.sectorName, this.description});

  Sector.fromJson(Map data) {
    this.sectorId = data['sectorId'];
    this.sectorName = data['sectorName'];
    this.description = data['description'];
  }
  Map<String, String> toJson() => <String, String>{
        'sectorId': sectorId,
        'sectorName': sectorName,
        'description': description,
      };
}
